src/book.py

```diff
@@ -125,7 +161,7 @@ def clean(self, lemmatize=True, deromanize=False, lemma_pos='v', inplace=False):
         if inplace:
             self._text = cleaned
         else:
-            return BookText(filepath=None, rawtext=cleaned, author=self.author, title=self.title, meta=self.meta)
+            return BookText(filepath=None, rawtext=cleaned, author=self._author, title=self._title, meta=self._meta)
```

Comment:
Doesn't it work either way? I haven't run into any issues here.

Reply:
Yes, it works either way.